src/components/OfferMyAccountCard/OfferMyAccountCard.tsx

```diff
@@ -67,6 +72,15 @@ const OfferMyAccountCard = ({ offerId }: OfferMyAccountCardProps) => {
   } = useAppSelector(selectOffer);
   const priceRules = price?.rules;
 
+  const getCustomerSwitches = async () => {
```

Comment:
for code cleanliness, please move this inside the `useEffect`. in the current form, to be consistent with the rules of hooks - it should be a dependency of the `useEffect`, which would cause an infinite re-render loop